sklearn/model_selection/_split.py

```diff
@@ -913,6 +920,170 @@ def get_n_splits(self, X, y, groups):
         return int(comb(len(np.unique(groups)), self.n_groups, exact=True))
 
 
+class _RepeatedSplits(with_metaclass(ABCMeta)):
+    """Repeated splits for an arbitrary randomized CV splitter.
+
+    Repeats splits for cross-validators n times with different randomization
+    in each repetition.
+
+    Parameters
+    ----------
+    cv : callable
+        Constructor of cross-validator.
```

Comment:
Isn't this the cross validation class itself? That seems more natural than passing the ``__init__`` method.